Corona-Warn-App/src/test/java/de/rki/coronawarnapp/service/submission/SubmissionServiceTest.kt

```diff
@@ -171,11 +171,17 @@ class SubmissionServiceTest {
             equalTo(true)
         )
 
-        // invalid
+        // invalid if no GUID
         assertThat(
             SubmissionService.containsValidGUID("https://no-guid-here"),
             equalTo(false)
         )
+
+        // invalid if url format is incorrect
+        assertThat(
+            SubmissionService.containsValidGUID("htps://wrongformat.com"),
```

Comment:
Can you use our new assertion framework here? I think it's already available in `dev`, try:

`SubmissionService.containsValidGUID("htps://wrongformat.com") shouldBe false`

Reply:
it's been done.